src/templates/_elements/toolbar.twig

```diff
@@ -36,7 +36,7 @@
                 {% endif %}
             </ul>
             {% if isAdministrative %}
-                {% if showStatusMenu %}<hr class="padded">{% endif %}
+                {% if showStatusMenu %}<hr class="padded" aria-hidden="true">{% endif %}
```

Comment:
@gcamacho079 not sure this is the best fix for [PT-80](https://linear.app/craftcms/issue/PT-80/the-status-menu-listbox-contains-children-roles-that-are-not-allowed) but it works.

Reply:
Great solution! 👍🏼 It’s a subtle difference, but `role="presentation"` also quiets the error while saying: “Ignore the usual semantics here.”

Reply:
@gcamacho079 is one better than the other? I'm happy to switch it to `role="presentation"` if that's more desirable.

Reply:
Yes, let's switch that over! Appreciation!